std-bits/image/src/main/java/org/enso/image/opencv/OpenCV.java

```diff
@@ -272,6 +272,7 @@ private boolean openCVNotFoundInJavaLibraryPath(String errorMessage) {
 
     /** Cleans up patches done to the environment. */
     @Override
+    @SuppressWarnings("deprecation")
```

Comment:
I'd not suppress deprecations. It can become unexpected error at some point

Can you please put finalization logic in the shutdown hook instead. It should be reasonable alternative given our usecase.

Reply:
I wanted to do that but then I noticed the header saying
```
// Copied unmodified from
// https://github.com/openpnp/opencv/blob/v4.5.1-0/src/main/java/nu/pattern/OpenCV.java
// to allow library loading on GraalVM.
```
So, I wanted to deal with this with a minimal amount of changes. @jdunkerley @kustosz can I change this file or what's the deal with this file? Also, shouldn't we incorporate copyright/license info here (seems to be deliberately missed)?

Reply:
Turned out we can remove that file all together.